app/src/main/kotlin/pl/gov/mc/protego/ui/main/HistoryViewModel.kt

```diff
@@ -0,0 +1,49 @@
+package pl.gov.mc.protego.ui.main
+
+import android.content.Intent
+import android.content.res.Resources
+import android.net.Uri
+import androidx.annotation.StringRes
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.polidea.cockpit.cockpit.Cockpit
+import pl.gov.mc.protego.R
+import pl.gov.mc.protego.information.AppInformation
+import pl.gov.mc.protego.ui.Event
+import pl.gov.mc.protego.util.EmailClientAdapter
+
+data class IntentToLaunch(
+    val intent: Intent,
+    @StringRes val errorMessage: Int
+)
+
+class HistoryViewModel(
+    private val appInformation: AppInformation,
+    private val resources: Resources,
+    private val emailClientAdapter: EmailClientAdapter
+) : ViewModel() {
+
+    val versionLiveData = MutableLiveData<String>()
+    val intentToStart = MutableLiveData<Event<IntentToLaunch>>()
```

Comment:
`MutableLiveData` wystawiane w publicznym interfejsie ViewModelu to nie jest dobra praktyka. W tym ViewModelu jest to zrobione lepiej: https://github.com/ProteGO-app/android/blob/master/app/src/main/kotlin/pl/gov/mc/protego/ui/registration/onboarding/OnboardingViewModel.kt#L55